gems/aws-sdk-s3/lib/aws-sdk-s3/plugins/s3_signer.rb

```diff
@@ -18,7 +17,6 @@ class S3Signer < Seahorse::Client::Plugin
 
         option(:sigv4_region) do |cfg|
           raise Aws::Errors::MissingRegionError if cfg.region.nil?
-
```

Comment:
This is a Rubocop suggestion, to have new lines following `raise` or `next` etc.

Reply:
Got updated by a rubocop -a.  I'll add the extra line back in.